core/core-parent/pom.xml

```diff
@@ -617,6 +617,7 @@
                                     <exclude>io.opentelemetry.extension</exclude>
                                     <exclude>io.opentelemetry.instrumentation</exclude>
                                     <exclude>fish.payara.shaded</exclude>
+                                    <exclude>org.glassfish.grizzly.http</exclude>
```

Comment:
We shouldn't ever be adding anything to this.
Please rework the Grizzly change

```suggestion
```

Reply:
already fixed